lua/lspconfig/server_configurations/markdown_oxide.lua

```diff
@@ -0,0 +1,18 @@
+return {
+  default_config = {
+    root_dir = function()
```

Comment:
The language server uses root_dir in the initialization, so right now single_file_support does not work. 

I do think that root_dir makes sense as is because this way the context in which the file is opened is sent as the root_dir, which makes sense for vaults with folders. The same root dir will be sent if neovim is called with `nvim .`, `nvim file`, and `nvim folder/file`, which adhears to the vault structure and the LSP completion structure

Reply:
if you set the single_file_support the root dir is cwd

Reply:
As many people do track their vaults with git, I think it is best to both resolve .git as root and cwd; I think root_dir is best. 

My recent commits have this